app/src/main/java/org/openobservatory/ooniprobe/common/MKCollectorResubmitTask.java

```diff
@@ -84,6 +84,8 @@ protected Void doInBackground(Integer... params) {
             throw new IllegalArgumentException("MKCollectorResubmitTask requires 2 nullable params: result_id, measurement_id");
         Where<Measurement> msmQuery = SQLite.select().from(Measurement.class)
                 .where(Measurement_Table.is_failed.eq(false))
+                .and(Measurement_Table.is_rerun.eq(false))
+                .and(Measurement_Table.is_done.eq(true))
```

Comment:
Please, check the documentation of the library we're using and make sure that this usage where we chain `.and` is equivalent to the usage where there are several arguments passed to `.where` (it should).

If that is true, then we should factor duplicated code more aggressively.

Reply:
I would recommend also unifying the code in here (or is there a reason not to?)